<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="mapper_awaits_before_reconnect" skipped="0" tests="1" time="7.35" timestamp="2022-04-24T12:15:55">
	<testcase classname="MapperReconnectAwait" name="mapper_awaits_before_reconnect">
		<failure message="FAILED: Line count on tedge_mapper_journal.out for &quot;::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0" type="FAILED"/>
		<system-out>12:15:48 INFO  ==============================================================
12:15:48 INFO  Id:    mapper_awaits_before_reconnect
12:15:48 INFO  Dir:   tedge_mapper_c8y/
12:15:48 INFO  Title: Validate mapper doesn't reconnect too often
12:15:48 INFO  ==============================================================
12:15:48 INFO  Started mosquitto_sub with process id 18366
12:15:48 INFO  Waiting for 0.1 seconds
12:15:49 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
12:15:49 INFO  Waiting for 5.0 seconds
12:15:54 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
12:15:55 INFO  Executed sudo&lt;tedge_mapper_journal&gt;, exit status 0
12:15:55 WARN  Grep on tedge_mapper_journal.out contains &quot;ERROR&quot; ... failed [run.py:67]
12:15:55 INFO  Line count on file tedge_mapper_journal.out ... passed
12:15:55 WARN  Line count on tedge_mapper_journal.out for &quot;::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0 ... failed [run.py:78]
12:15:55 WARN  Aborted test due to FAILED outcome
12:15:55 INFO  
12:15:55 INFO  Test duration: 7.35 secs
12:15:55 INFO  Test final outcome:  FAILED
12:15:55 INFO  Test outcome reason: Line count on tedge_mapper_journal.out for &quot;::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0
12:15:55 INFO  
</system-out>
	</testcase>
</testsuite>
